README.md

```diff
@@ -0,0 +1,75 @@
+Branding
+========
+
+Project brand assets and guidelines for ReactPHP.
+
+Logo
+----
+
+| Name         | SVG                                                                      | PNG                                                                      |
+|:-------------|:-------------------------------------------------------------------------|:-------------------------------------------------------------------------|
+| Logo         | [![Logo][img-logo-svg]][link-logo-svg]                                   | [![Logo][img-logo-png]][link-logo-png]                                   |
+| Symbol       | [![Symbol][img-logo-svg-symbol]][link-logo-svg-symbol]                   | [![Symbol][img-logo-png-symbol]][link-logo-png-symbol]                   |
+| Symbol Shape | [![Symbol Shape][img-logo-svg-symbol-shape]][link-logo-svg-symbol-shape] | [![Symbol Shape][img-logo-png-symbol-shape]][link-logo-png-symbol-shape] |
+| Type         | [![Type][img-logo-svg-type]][link-logo-svg-type]                         | [![Type][img-logo-png-type]][link-logo-png-type]                         |
+| Icon         | [![Icon][img-logo-svg-icon]][link-logo-svg-icon]                         | [![Icon][img-logo-png-icon]][link-logo-png-icon]                         |
+
+[img-logo-svg]: https://rawgit.com/reactphp/branding/master/preview/reactphp-logo.svg
+[link-logo-svg]: reactphp-logo.svg
+[img-logo-png]: https://rawgit.com/reactphp/branding/master/preview/reactphp-logo.png
+[link-logo-png]: reactphp-logo.png
+
+[img-logo-svg-symbol]: https://rawgit.com/reactphp/branding/master/preview/reactphp-symbol.svg
+[link-logo-svg-symbol]: reactphp-symbol.svg
+[img-logo-png-symbol]: https://rawgit.com/reactphp/branding/master/preview/reactphp-symbol.png
+[link-logo-png-symbol]: reactphp-symbol.png
+
+[img-logo-svg-symbol-shape]: https://rawgit.com/reactphp/branding/master/preview/reactphp-symbol-shape.svg
+[link-logo-svg-symbol-shape]: reactphp-symbol-shape.svg
+[img-logo-png-symbol-shape]: https://rawgit.com/reactphp/branding/master/preview/reactphp-symbol-shape.png
+[link-logo-png-symbol-shape]: reactphp-symbol-shape.png
+
+[img-logo-svg-type]: https://rawgit.com/reactphp/branding/master/preview/reactphp-type.svg
+[link-logo-svg-type]: reactphp-type.svg
+[img-logo-png-type]: https://rawgit.com/reactphp/branding/master/preview/reactphp-type.png
+[link-logo-png-type]: reactphp-type.png
+
+[img-logo-svg-icon]: https://rawgit.com/reactphp/branding/master/preview/reactphp-icon.svg
+[link-logo-svg-icon]: reactphp-icon.svg
+[img-logo-png-icon]: https://rawgit.com/reactphp/branding/master/preview/reactphp-icon.png
+[link-logo-png-icon]: reactphp-icon.png
+
+Color Palette
```

Comment:
I think it helps to be explicit about the color palette that we use for our *branding* here :+1: 

In other words: IMHO we should keep the colors that define our logo etc. and use relevant names here, such as "text foreground", "logo background" etc.

Arguably, the website is not part of the branding, so I agree it's probably safer to exclude those colors here :+1: